docs/source/gettingstarted.rst

```diff
@@ -61,7 +61,7 @@ version number.
 .. code-block:: text
 
     $ sqlfluff version
-    0.3.1
+    1.2.1
```

Comment:
@tunetheweb, @alanmcruickshank: How difficult would it be to automatically generate/update this as part of the release process?

Reply:
@greg-finley ?

Reply:
Looks easy, I can send it over in a few hours.